frontend/ui/src/app/components/installation-wizard/installation-wizard.component.html

```diff
@@ -266,6 +266,19 @@ <h3 class="font-normal text-gray-900 dark:text-white">Deploy</h3>
                       </app-yaml-editor>
                     </div>
                   }
+                  @if (deployForm.controls.envFileData.enabled) {
+                    <div class="col-span-2">
+                      <label for="env-file" class="block mb-2 text-sm font-medium text-gray-900 dark:text-white">
+                        Environment values
+                      </label>
+
+                      <app-yaml-editor
```

Comment:
Using YAML syntax highlighting here might be confusing since some incorrect syntax might look more "correct" in the editor than the correct syntax. At the same time, it doesn't look like changing the installed extensions on the fly is particularly easy so we would probably need a separate component without syntax highlighting for that.

So, since I think it's probably not a problem for now, we can leave as is but I would like to change this to an editor view without syntax highlighting in the future